developer-docs-site/docs/whats-new-in-docs.md

```diff
@@ -7,6 +7,10 @@ slug: "whats-new-in-docs"
 
 This page shows the key updates to the developer documentation on this site. Note, this site is built from the `main` upstream branch of GitHub and so therefore reflects the latest changes to Aptos. If you have checked out [another branch](https://github.com/aptos-labs/aptos-core/branches) to use a [specific network](guides/system-integrators-guide.md#choose-a-network), the code may not yet have all of the features described here.
 
+## 28 March 2023
+
+- Offered an experimental feature to [profile gas use locally](./cli-tools/aptos-cli-tool/use-aptos-cli.md#profiling-gas-use) via the Aptos CLI that not only simulates usage but also generates flame graphs for visualizing transaction gas logs.
```

Comment:
Same here, maybe remove the reference to "transaction gas logs"?

Reply:
Done!